<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceDialogClass</class>
 <widget class="QDialog" name="InterfaceDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterfaceDialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>480</width>
     <height>380</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Type</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>375</width>
       <height>270</height>
      </rect>
     </property>
     <property name="title">
      <string>Choose a Name</string>
     </property>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IF name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cInterfaceName">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>25</y>
        <width>250</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="cDescription">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>250</width>
        <height>64</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="cDelete">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>155</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cCopy">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>125</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cNew">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>95</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Prompts</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>136</height>
      </rect>
     </property>
     <property name="title">
      <string>Prompts</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Command</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Timeout</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Connected</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cPromptCommand">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>25</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cPromptTimeout">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cPromptConnected">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>75</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnected</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cPromptDisconnected">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>TNC Cmds</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Commands</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Mycall</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Retry</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Convers</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>DayTime</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cCommandMyCall">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>25</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cCommandConnect">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cCommandRetry">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>75</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cCommandConvers">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cCommandDayTime">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>125</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>450</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>TNC Command Prefix</string>
     </property>
     <widget class="QCheckBox" name="cIncludeCommandPrefix">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>210</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Include TNC command prefix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cCommandPrefix">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>35</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>30</y>
        <width>124</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>(single number only)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Init Cmds</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>296</height>
      </rect>
     </property>
     <property name="title">
      <string>Initialization Commands</string>
     </property>
     <widget class="QRadioButton" name="cNeverSendInitCommands">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>450</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Never send TNC initalization commands</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="cAlwaysSendInitCommands">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>450</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Send TNC initialization commands for every Send/Recevive (S/R) session</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>85</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cmds to send before S/R</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>85</y>
        <width>150</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cmds to send after S/R</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="cCommandsBefore">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>150</width>
        <height>170</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="cCommandsAfter">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>110</y>
        <width>150</width>
        <height>170</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Com Port</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>196</height>
      </rect>
     </property>
     <property name="title">
      <string>Comm Port Settings</string>
     </property>
     <widget class="QComboBox" name="cStopBits">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>125</y>
        <width>90</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cComPort">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>25</y>
        <width>325</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>80</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Com port</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cBaud">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>90</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>110</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>256000</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cDataBits">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>90</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cParity">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>75</y>
        <width>90</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>130</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop bits</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>55</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Baud</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>105</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data bits</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cFlowControl">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>90</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RTS/CTS</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>155</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Flow control</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>AGWPE</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>310</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Telnet</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>310</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cOK">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>395</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cCancel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>395</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cFlowControl</tabstop>
  <tabstop>cPromptCommand</tabstop>
  <tabstop>cPromptTimeout</tabstop>
  <tabstop>cPromptConnected</tabstop>
  <tabstop>cPromptDisconnected</tabstop>
  <tabstop>cCommandMyCall</tabstop>
  <tabstop>cCommandConnect</tabstop>
  <tabstop>cCommandRetry</tabstop>
  <tabstop>cCommandConvers</tabstop>
  <tabstop>cCommandDayTime</tabstop>
  <tabstop>cNeverSendInitCommands</tabstop>
  <tabstop>cAlwaysSendInitCommands</tabstop>
  <tabstop>cComPort</tabstop>
  <tabstop>cBaud</tabstop>
  <tabstop>cParity</tabstop>
  <tabstop>cDataBits</tabstop>
  <tabstop>cStopBits</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cOK</sender>
   <signal>clicked()</signal>
   <receiver>InterfaceDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cCancel</sender>
   <signal>clicked()</signal>
   <receiver>InterfaceDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
